anchorecli/cli/utils.py

```diff
@@ -934,6 +931,22 @@ def format_vulnerabilities(payload, params):
     return obuf
 
 
+def format_content_query(payload):
+    content = payload.get('content', '')
+    if not content:
+        return ''
+    if isinstance(content, list):
+        # In some situations the `content` key can be a list, not a string
+        content = ''.join(content)
```

Comment:
for presentation should we join by `', '` instead of `''` ?

Reply:
This is coming from the empty list though, and in the event that it does have content... are you expecting every line to be an item? Or a single string in the list? From what we observed it was always an empty list

Reply:
Actually, I didn't know what to expect --though I do recall the empty list case from yesterday. It seems that the API could return a content list with multiple items (that is, it is not precluded from happening here). Wasn't certain what the right call for presentation of multiple items in a list here would be (`', '.join()` or `'\n'.join()` maybe?). 

If we're certain that the given content is always empty, then this comment can safely be ignored :)

Reply:
If anything happens that doesn't conform to what we expect we can update this. Thanks for the detail in the review!